crawl-ref/source/dat/des/portals/bazaar.des

```diff
@@ -1548,6 +1548,65 @@ xT.xxxxxxxxxxx.Tx
 xxxxxxxxxxxxxxxxx
 ENDMAP
 
+# 3 to 4 shops, with an average of 3.5
+NAME:       bazaar_kenran_forest_clearing
+TAGS:       no_rotate no_vmirror
+ORIENT:     encompass
+WEIGHT:     5
+KITEM:      h = human skeleton
+KFEAT:      $ = any shop
+KFEAT:      # = antique weapon shop / antique armour shop / \
+                antiques shop / book shop
+KFEAT:      A = abandoned_shop
+: if crawl.one_chance_in(4) then
+KFEAT:      F = V
+TILE:       t = dngn_tree_dead w:10 / dngn_tree_red w:3 / \
+                dngn_tree_lightred w:2
+COLOUR:     . = brown, t = brown w:10 / red w:3 / lightred w:2
+: else
+TILE:       t = dngn_tree w:3 / dngn_tree_yellow w:2 / \
+                dngn_tree_lightred w:2 / dngn_tree_red w:3
+KFEAT:      F = T
+COLOUR:     . = green, t = green w:3 / lightgreen w:2 / \
+                lightred w:3 / red w:2 / brown w:1
+: end
+NSUBST:     $ = 1=$A
+SUBST:      $ = #:20 $
+:           bazaar_setup(_G)
+MAP
+ttttttttttttttttttttttttt
+tttttttttttttttttttttt>tt
+ttttttttttttttttttttt.ttt
+ttttttttttttttttttttt.ttt
+ttttttttttttttGttt.t..ttt
```

Comment:
I don't like the location of this G: it means you never get to see the two trees NE and SW of it, which leaves the map feeling a little unexplored. More generally I don't like transparent-but-impassable obstacles that are at a 90-degree corner between two walls.

This would be fine if it were moved south a tile, or east, or just put somewhere else where it's not in a corner.